app/screens/AddEmission/AddEmissionScreen.navigationOptions.tsx

```diff
@@ -1,33 +1,18 @@
 import React from "react";
-import { Ionicons } from "@expo/vector-icons";
-
-import { useNavigation } from "@react-navigation/native";
 
 import { Text } from "../../components";
 import { t } from "../../utils";
 import { Colors, ComponentsStyle } from "../../style";
-import { navigate } from "../../navigation";
+import { InfoButton } from "./components";
 
-const navigationOptions = () => {
-  const navigation = useNavigation();
```

Comment:
after adding the recommended hook rules lint rule, this hook call was shown to be used outside of a React component, which caused the warnings. I have extracted the hook call and its usage into a separate component.